pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -723,4 +738,12 @@ func printCheckWrapper(cw *CheckWrapper) {
 	logrus.Debugf("node_type: %+v", cw.NodeType)
 	logrus.Debugf("nodes: %+v", cw.Nodes)
 	logrus.Debugf("result: %+v", cw.Result)
+	logrus.Debugf("auditConfig: %+v", cw.AuditConfig)
+	logrus.Debugf("audit: %+v", cw.Audit)
+	logrus.Debugf("commands: %+v", cw.Commands)
+	logrus.Debugf("configcommands: %+v", cw.ConfigCommands)
+	logrus.Debugf("testInfo: %+v", cw.TestInfo)
+	logrus.Debugf("ActualValue: %v", cw.ActualValue)
+	logrus.Debugf("ExpectedResult: %v", cw.ExpectedResult)
```

Comment:
I realize there is a precedent for this pattern but it seems odd to me. Why not use something like [string builder](https://golang.org/pkg/strings/#Builder) to build the debug log and print that? Or just marshall the object and print the result?

Reply:
I can change this, but actually no where in the cis scan tooling is this code invoked really

Reply:
I just appended extra fields I am adding - the debug is for anyone who will run this go tool like cli separately.

Reply:
Oh, could just do a `%+v ` for the whole object then? No need to change it now

Reply:
yeah makes sense, changed it to use %+v